src/transformers/models/perceiver/modeling_perceiver.py

```diff
@@ -857,7 +869,9 @@ def forward(
         return_dict = return_dict if return_dict is not None else self.config.use_return_dict
 
         if self.input_preprocessor is not None:
-            inputs, modality_sizes, inputs_without_pos = self.input_preprocessor(inputs)
+            inputs, modality_sizes, inputs_without_pos = self.input_preprocessor(
+                inputs, interpolate_pos_encoding=interpolate_pos_encoding
+            )
```

Comment:
@amyeroberts since `self.input_processor` here could also be either `TextPreprocessor` or `MultimodalPreprosessor`, it would complain `got unexpected keyword argument` if it was not added to the `forward()` of those two corresponding methods. Alternatively, we could add another condition here to call `self.input_processor` with two arguments unless it is `ImagePreprocessor`

Reply:
No, that's OK, I just wanted to make sure that this was deliberate